clients/java/signalr/src/test/java/com/microsoft/signalr/TestHttpClient.java

```diff
@@ -66,4 +65,8 @@ public TestHttpClient on(String method, String url, Function<HttpRequest, Single
     public WebSocketWrapper createWebSocket(String url, Map<String, String> headers) {
         throw new RuntimeException("WebSockets isn't supported in testing currently.");
     }
+
+    interface TestHttpClientHandler {
```

Comment:
A slightly more descriptive name (since it is a function that handles requests) suggestion:

```suggestion
    interface TestHttpRequestHandler {
```